companion/src/firmwares/opentx/opentxinterface.cpp

```diff
@@ -763,14 +763,14 @@ bool OpenTxFirmware::isAvailable(PulsesProtocol proto, int port)
           case PULSES_GHOST:
             return true;
           case PULSES_ACCESS_R9M:
-            return IS_TARANIS_XLITE(board) || IS_TARANIS_X9LITE(board) || board == BOARD_TARANIS_X9DP_2019 || board == BOARD_X10_EXPRESS || (IS_FAMILY_HORUS_OR_T16(board) && id.contains("internalaccess"));
+            return IS_TARANIS_XLITE(board) || IS_TARANIS_X7_ACCESS(board) || IS_TARANIS_X9LITE(board) || board == BOARD_TARANIS_X9DP_2019 || board == BOARD_X10_EXPRESS || (IS_FAMILY_HORUS_OR_T16(board) && id.contains("internalaccess"));
```

Comment:
maybe return` IS_TARANIS_XLITE(board) || IS_ACCESS_RADIO(board) `?

Reply:
I didn't think all ACCESS radios have an external module and thus why I chose the specific radio here

Reply:
...or maybe if there are a few exceptions return (IS_ACCESS_RADIO(board, id) && !IS_radiox && !IS_radioy)?

Reply:
I checked and all seem to accept it on the radio, even things like xlites, adapters do exist and soem prefer the larger modules

Reply:
Thanks for the info so we'll go with your recommendation